docs/rules/indent.md

```diff
@@ -35,12 +35,12 @@ The `indent` rule has two options:
 
 Level of indentation denotes the multiple of the indent specified. Example:
 
-* Indent of 4 spaces with `VariableDeclarator` set to 2 will indent the multi-line variable declarations with 8 spaces.
-* Indent of 2 spaces with `VariableDeclarator` set to 2 will indent the multi-line variable declarations with 4 spaces.
+* Indent of 4 spaces with `VariableDeclarator` set to 3 will indent the multi-line variable declarations with 6 spaces with respect to the variable declarator keyword.
```

Comment:
I do not think that's correct. Indent of 4 spaces with `VariableDeclarator` will indent by 12 spaces (3x4)